cpp_test_suite/new_tests/cxx_nan_inf_in_prop.cpp

```diff
@@ -5,6 +5,15 @@
 #include <cxxtest/TangoPrinter.h>
 #include <tango.h>
 #include <iostream>
+#ifdef _WIN32
+#include <limits>
+#ifndef isinf
+#define isinf(x) ((x)!=(x))
+#endif
+#ifndef isnan 
+#define isnan(x) ((x)!=(x)) 
+#endif
+#endif
```

Comment:
First of all NaN and infs are not the same.

Second recent enough VS versions have std::isnan and std::isinf defined (since C++11), so no need for supplying them for these versions.

For older VS versions I would use something like [1].

[1]: https://stackoverflow.com/a/20723890